src-docs/src/views/badge/badge_example.js

```diff
@@ -188,6 +200,37 @@ export const BadgeExample = {
       snippet: badgeButtonSnippet,
       demo: <BadgeButton />,
     },
+    {
+      title: 'Badge for health status',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: badgeHealthSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: badgeHealthHtml,
+        },
+      ],
+      text: (
+        <div>
+          <p>
+            Badges can work as health status indicators in places where there
+            are a lot of repeated statuses e.g. in tables.
```

Comment:
```suggestion
            are a lot of repeated statuses, e.g. in tables.
```

I think keeping this example to strictly show the badges is the right call. We tend not to try to complicate small display component examples with larger ones.

That said, we **could**, add a column to one of the EuiTable examples that shows using the badges in a column which we could then link to from here. I think we currently show EuiHealth in most of them.

Reply:
Ah true, that might be a better example to give. I'll see if I can put that together for this PR as well.

Reply:
I've updated the custom table example (https://github.com/elastic/eui/pull/4006/commits/0c2aa9290b5180d0d0dda90ecb5a6ed92716d472) to use `EuiBadge` for the status and updated the "new" badge.